cmd/teamcity-trigger/main.go

```diff
@@ -29,13 +29,16 @@ func main() {
 	importPaths := gotool.ImportPaths([]string{"github.com/cockroachdb/cockroach/..."})
 
 	client := teamcity.New("teamcity.cockroachdb.com", username, password)
-	for _, importPath := range importPaths {
-		build, err := client.QueueBuild(*buildTypeID, *branchName, map[string]string{
-			"env.PKG": importPath,
-		})
-		if err != nil {
-			log.Fatalf("failed to create teamcity build for %s: %s", importPath, err)
+	for _, goflagsVal := range []string{"-race", "-tags deadlock"} {
+		for _, importPath := range importPaths {
+			build, err := client.QueueBuild(*buildTypeID, *branchName, map[string]string{
+				"env.PKG":     importPath,
+				"env.GOFLAGS": goflagsVal,
+			})
+			if err != nil {
+				log.Fatalf("failed to create teamcity build for %s: %s", importPath, err)
+			}
+			log.Printf("created teamcity build for %s: %s", importPath, build)
```

Comment:
Does this should that a race/deadlock build is being performed? (I'm too lazy to see what `build` contains).